### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # bedrock-vue-organization-form ChangeLog
 
+## 2.0.0 -
```
Comment (davidlehn) [inline CHANGELOG.md:3]: Merge the 1.1.0 section below into this if that's not going to be released.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 
 ### Added
-- Added new entity types.
+- Add new entity types.
+- Add Eslint and dependencies to the project.
+
+### Removed
+- Deprecate components such as q-layout-header.
+- Deprecate css classes such as gutter-md.
```
Comment (dlongley) [inline CHANGELOG.md]: If these are only deprecated and not fully removed, let's put it under `Changed`

Comment (aljones15) [reply CHANGELOG.md]: their names were changed actually.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
+### Changed
+- Upgrade bedrock-quasar to version 3.0.0.
+- Upgrade Font Awesome to version 5.
+- Change karma config Quasar umd to the latest umd.
 
 ## 1.0.0 - 2019-02-27
 
 ### Added
 - Add core files.
-
```
Comment (dlongley) [inline CHANGELOG.md]: Keep this space ... that last comment (about git history) isn't an "Added" comment. We've got it in many of our changelogs as just a standalone comment at the end.

File: CHANGELOG.md (hunk 2 above)
Comment (dlongley) [reply CHANGELOG.md]: Then they should be under `Changed` and say ```***BREAKING***: Changed X to Y.``` for each one.

Comment (aljones15) [reply CHANGELOG.md]: I changed it - Remove deprecated components such as q-layout-header.

Comment (dlongley) [reply CHANGELOG.md]: Ok, that makes more sense! +1

Comment (aljones15) [reply CHANGELOG.md]: lol then I changed to - **BREAKING** anyways what's there is ok.

**Pull request merged**